redaxo/src/core/lib/response.php

```diff
@@ -226,14 +226,15 @@ public static function sendFile($file, $contentType, $contentDisposition = 'inli
      * @param null|string $etag               HTTP Cachekey to identify the cache
      * @param null|string $contentDisposition Content disposition ("inline" or "attachment")
      * @param null|string $filename           Filename
+     * @param null|string $cacheControl       Cache Control
      */
-    public static function sendResource($content, $contentType = null, $lastModified = null, $etag = null, $contentDisposition = null, $filename = null)
+    public static function sendResource($content, $contentType = null, $lastModified = null, $etag = null, $contentDisposition = null, $filename = null, $cacheControl = 'max-age=3600, must-revalidate, proxy-revalidate, private')
```

Comment:
Ich würde das lieber nicht als zusätzlichen Parameter hinzufügen, sondern stattdessen in der Methode um den Aufruf ein if packen so wie in sendFile:

https://github.com/redaxo/redaxo/blob/9d4ec61f1733c80d6623759e0018e65180732fff/redaxo/src/core/lib/response.php#L170-L172

Damit wird dann abgefangen, wenn vorher selbst `rex_response::sendCacheControl()` aufgerufen wurde.

Reply:
OK, bleibt die Frage nach dem default-Wert.